compiler/plugins/target/ROCM/test/opt_pass_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+set(GPU_INSTRUMENTATION_PASSES
```

Comment:
This will need hardening to be in-tree - it needs to support all platforms at minimum. We try to avoid arbitrary cmake and stick to a more structured form as seen in other directories - this handles the goo for cross-platform/cross-compilation support and ensures things work. If this is intended as a sample then we can move it someplace else - but if it's a test then it needs to build and run everywhere the compiler does.

Reply:
If I use iree_cc_library() instead, will that work?